src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine{

```

Comment:
There should be a space between the superclass name and the curly brace to follow Java code conventions: `public class Truck extends Machine {`.